Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM tymkillerapps/nodejs-arch:latest
+MAINTAINER Tyler Poush <hooch0022@gmail.com>
+RUN mkdir /mnt/videos
+WORKDIR /opt
+RUN git clone https://github.com/criticalfault/HomeFlix.git /opt/HomeFlix
```

Comment:
So, I don't see much of a reason to have this container clone down the repo, since if we are going to put it on DockerHub, the container will already contain all the files it needs (see my PR). Even if they wanted to clone the repo themselves locally, with the build script I have written (again, its super minimal), all they would have to do is run that (and include the file override like i mention in a different comment) and it will build the docker container for them with all project files.